src/python-common/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(Distutils)
+distutils_install_module(ceph
+  INSTALL_SCRIPT ${CMAKE_INSTALL_FULL_SBINDIR})
```

Comment:
```
Copying ceph.egg-info to /home/jenkins-build/build/workspace/ceph-dev-new-build/ARCH/x86_64/AVAILABLE_ARCH/x86_64/AVAILABLE_DIST/centos7/DIST/centos7/MACHINE_SIZE/huge/release/15.0.0-3060-g1d49401/rpm/el7/BUILDROOT/ceph-15.0.0-3060.g1d49401.el7.x86_64/usr/lib/python3.6/site-packages/ceph-1.0.0-py3.6.egg-info
...
File not found by glob: /home/jenkins-build/build/workspace/ceph-dev-new-build/ARCH/x86_64/AVAILABLE_ARCH/x86_64/AVAILABLE_DIST/centos7/DIST/centos7/MACHINE_SIZE/huge/release/15.0.0-3060-g1d49401/rpm/el7/BUILDROOT/ceph-15.0.0-3060.g1d49401.el7.x86_64/usr/lib/python2.7/site-packages/ceph-*.egg-info
```

`distutils_install_module()` install a python package using python3 by default. you need to do something like 

https://github.com/ceph/ceph/blob/ed6a047c71bf430c8f706d3c9b064b33533cae7b/src/ceph-volume/CMakeLists.txt#L4-L12

sorry @sebastian-philipp , this change was made recently. i wanted to make Ceph more python3 friendly, that's why i made `distutils_install_module()` default to python3, and let the caller to decide which python to use.

Reply:
keeping Python 2 support is really painful.

Reply:
wondering if that would help in making a python-ceph-common library. Unfortunately no, because my problem originated in the RPM spec.

Reply:
actually, you can. but i wonder why we need to do so. as IIUC, the major consumer of this python module will be mgr modules. and mgr can only embed a single version of python which is specified at compile time.

Reply:
let's give it a try tomorrow